ui/yarn.lock

```diff
@@ -2256,22 +2256,22 @@
     "@webassemblyjs/wast-parser" "1.9.0"
     "@xtuc/long" "4.2.2"
 
-"@webpack-cli/configtest@^1.0.3":
```

Comment:
not sure if lock file needs to be committed. @alexmt is it fine to commit this file?

Reply:
@harikrongali Lock file does need to be committed, but we should avoid bumping versions in any of these packages if possible. @schakrad I don't see any code changes that should require updating package versions, can you confirm?

Reply:
Thanks, @rbreeze for confirmation. @schakrad please revert the package.json update

Reply:
Sure Hari, I will check and revert back.